src/FixedDataTableCellGroup.js

```diff
@@ -213,13 +213,18 @@ class FixedDataTableCellGroup extends React.Component {
 
     var style = {
       height: props.cellGroupWrapperHeight || props.height,
-      width: props.width
+      width: props.width,
+      left: offsetLeft,
     };
 
-    if (DIR_SIGN === 1) {
-      style.left = offsetLeft;
-    } else {
-      style.right = offsetLeft;
+    if (Locale.isRTL()) {
+      if (props.fixedRight) {
```

Comment:
@louiebert, can you explain why this was needed?

Reply:
Turns out it wasn't. I was doing an unnecessary precalculation on the offsetLeft prop. Good catch, thanks!